test/integration/next-image-new/app-dir/app/static-img/page.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import testImg from '../../public/foo/test-rect.jpg'
+import testImg from '../../public/foo/test-rect-broken.jpg'
```

Comment:
Is this intentional?

Reply:
Weird. Looks like this line was overritten by some test script and then somehow I've commited it. Fixing it now